scripts/deploy-staking-token.js

```diff
@@ -0,0 +1,96 @@
+// Assumes network is started
+
+const Web3 = require('web3');
+const fs = require('fs');
+const path = require('path');
+const solc = require('solc');
+const constants = require('../utils/constants');
+const SnS = require('../utils/signAndSendTx.js');
+const web3 = new Web3('http://localhost:8541');
+const BN = web3.utils.BN;
+const ValidatorSetContract = require(path.join(__dirname, '../utils/getContract'))('ValidatorSetAuRa', web3);
+const OWNER = constants.OWNER;
+const expect = require('chai')
+    .use(require('chai-bn')(BN))
+    .use(require('chai-as-promised'))
+    .expect;
+const pp = require('../utils/prettyPrint');
+let tokenName = 'POSDAO';
+let tokenSymbol = 'POS';
+let tokenDecimals = 18;
+
+function compileContract() {
+    const web3 = new Web3('http://localhost:8541');
```

Comment:
Can't you just use `web3` defined in the same module above?

Reply:
Actually this line can be removed because `web3` is not used in this function.

Reply:
You're right, thanks